frontend/tests/pages/acceptInvitationPage.ts

```diff
@@ -0,0 +1,18 @@
+import type { Page } from '@playwright/test';
+import { BasePage } from './basePage';
+
+export class AcceptInvitationPage extends BasePage {
+  constructor(page: Page) {
+    super(page, page.getByRole('heading', {name: 'Accept Invitation'}), `/acceptInvitation`);
+  }
+
+  async fillForm(name: string, email: string, password: string): Promise<void> {
+    await this.page.getByLabel('Name').fill(name);
+    await this.page.getByLabel('Email').fill(email);
```

Comment:
this should be optional since the Accept invite page should have the email pre populated

Reply:
Done in commit 024113c1.